src/screens/ContactScreen/index.js

```diff
@@ -34,8 +34,12 @@ import useGroupInfo from '../GroupInfo/hooks/useGroupInfo';
 import {ANONYMOUS} from '../../hooks/core/constant';
 import {Button} from '../../components/Button';
 import {COLORS} from '../../utils/theme';
+import {
+  CONTACT_SCREEN_MAX_SELECTED_USERS,
+  DEFAULT_PROFILE_PIC_PATH,
+  NavigationConstants
+} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include a new constant `CONTACT_SCREEN_MAX_SELECTED_USERS`. This is a good practice as it improves code readability and maintainability by avoiding magic numbers in the code.
```diff
- import {DEFAULT_PROFILE_PIC_PATH, NavigationConstants} from '../../utils/constants';
+ import {
+   CONTACT_SCREEN_MAX_SELECTED_USERS,
+   DEFAULT_PROFILE_PIC_PATH,
+   NavigationConstants
+ } from '../../utils/constants';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->